src/Shared/FileUtilities.cs

```diff
@@ -266,12 +268,26 @@ internal static bool IsSlash(char c)
         /// <summary>
         /// Trims the string and removes any double quotes around it.
         /// </summary>
-        internal static string TrimAndStripAnyQuotes(string path)
+        ///
+        public static bool EnsurePathIsSingleQuoted(string path)
         {
-            // Trim returns the same string if trimming isn't needed
-            path = path.Trim();
-            path = path.Trim(new char[] { '"' });
+            int endId = path.Length - 1;
```

Comment:
The idea behind this was to not just check if `path` has a single quote as its first and last character but, if it does not, return path with an added quote on either end. So the return type should be string.